sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -258,20 +260,21 @@ export class EducationProgramOfferingController extends BaseController {
   @AllowAuthorizedParty(AuthorizedParties.institution)
   @HasLocationAccess("locationId")
   @Get(
-    "location/:locationId/education-program/:programId/program-year/:programYearId/offerings-list",
+    "location/:locationId/education-program/:programId/program-year/:programYearId/selectedOffering/:selectedOffering/offerings-list",
```

Comment:
looks like this API is for `offerings-list` for the dropdown. then why are we passing selectedOfferingId as a path parameter here? please let me know, if i am missing anything